src/symbols/tlaSymbols.ts

```diff
@@ -91,17 +119,46 @@ class ParsingContext {
 }
 
 /**
- * Provides TLA+ symbols from the given document.
+ * Provides TLA+ symbols from documents identified by URI.
+ * Converts fileUri into suitable parameters for the existing TlaDocumentSymbolsProvider
  */
-export class TlaDocumentSymbolsProvider implements vscode.DocumentSymbolProvider {
+export class TLADocumentSymbolProvider implements vscode.DocumentSymbolProvider {
+    private tlaDocSymbolsProvider: TlaDocumentSymbolsProvider;
+
     constructor(
         private readonly docInfos: TlaDocumentInfos
-    ) {}
+    ) {
+        this.tlaDocSymbolsProvider = new TlaDocumentSymbolsProvider(docInfos);
+    }
 
     provideDocumentSymbols(
         document: vscode.TextDocument,
         token: vscode.CancellationToken
     ): vscode.ProviderResult<vscode.SymbolInformation[] | vscode.DocumentSymbol[]> {
+        return this.tlaDocSymbolsProvider.provideDocumentSymbols(document, token);
```

Comment:
this function is async, shouldn't we have an await call here or is it safe to skip?